CONTRIBUTING.md

```diff
@@ -77,6 +77,31 @@ Or run the documentation and browse to [http://localhost:1234/](http://localhost
 yarn start:docs
 ```
 
+### Component/Hook scaffolding
+If you are looking to contribute a brand new component or hook in a package that does not exist yet, please run the following command.
+```bash
+yarn plop
+```
+This will start a series of cli prompts to determine what template files and folders should be generated to help quick start your contribution.
+The prompts are as follows:
+```
+1. What type of project are you setting up?
+- Select "React Spectrum v3"
+
+2. scope name(s)
```

Comment:
Matching the rest of the items in this tutorial, unless we want to copy the prompt headings verbatim as they all seem to start in lowercase.
```suggestion
2. Scope name(s)
```